tests/www/views/test_views_dagrun.py

```diff
@@ -61,16 +61,16 @@ def init_blank_dagrun():
     This really shouldn't be needed, but tests elsewhere leave the db dirty.
     """
     with create_session() as session:
-        session.query(DagRun).delete()
         session.query(TaskInstance).delete()
+        session.query(DagRun).delete()
```

Comment:
Should we also update https://github.com/apache/airflow/blob/92e775962dd74bcc350bdb0445e9729b706a2f2f/tests/test_utils/db.py#L47-L48

Reply:
I just did, was about to push 😂

Reply:
I also took the chance to just use `clear_db_runs` in the fixture as well. A few more db calls (but should not actually remove anything) for easier maintenance.